imagebuildah/build.go

```diff
@@ -61,6 +61,14 @@ type BuildOptions struct {
 	Args map[string]string
 	// Name of the image to write to.
 	Output string
+	// Log is a callback that will print a progress message.  If no value
+	// is supplied, the message will be sent to Err (or os.Stderr, if Err
+	// is nil) by default.
+	Log func(format string, args ...interface{})
+	// Out is a place where low-priority status messages are sent.
+	Out io.Writer
+	// Err is a place where higher-priority status messages are sent.
+	Err io.Writer
```

Comment:
Seems a bit weird to call this Err, if all it is getting is "higher-priority status messages"